run_server.py

```diff
@@ -3,5 +3,5 @@
 # settings enabled.
 from Donut import app, config
 
-port = getattr(config, 'PORT', 5000)
+port = getattr(config, 'PORT', 5050)
```

Comment:
If you're having issues with multiple people testing at once, you can individually change the port in the config file (`config.py` isn't part of the codebase so you can change it arbitrarily)